packages/material-ui/src/SnackbarContent/SnackbarContent.test.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import { assert } from 'chai';
-import { createShallow, createMount, getClasses } from '@material-ui/core/test-utils';
+import { expect } from 'chai';
+import { createClientRender } from 'test/utils/createClientRender';
+import { createMount, getClasses } from '@material-ui/core/test-utils';
 import describeConformance from '../test-utils/describeConformance';
 import Paper from '../Paper';
 import SnackbarContent from './SnackbarContent';
 
 describe('<SnackbarContent />', () => {
   let mount;
-  let shallow;
   let classes;
+  const render = createClientRender({ strict: false });
```

Comment:
```suggestion
  const render = createClientRender({ strict: true });
```
We try to run every test in `React.StrictMode` when possible.

Reply:
Updated!

I did notice the Select.test.js file has strict mode set to false. I'm not sure if that's intentional because it can't use strict mode, but just wanted to let you know in case it's not intentional.